presto-hive-common/src/main/java/com/facebook/presto/hive/rule/BaseSubfieldExtractionRewriter.java

```diff
@@ -485,7 +485,7 @@ public Void visitVariableReference(
         }
     }
 
-    public static boolean useDynamicFilter(
```

Comment:
Why is this needed? Why can't the downstream dependency use this static method?

Reply:
We need to override this method. At this moment it is just a placeholder static method that always return false. With static method it is not dynamic binding in Java.